.gitignore

```diff
@@ -6,4 +6,12 @@ gen/
 # sqlite
 **/*.db
 
-.*.env
\ No newline at end of file
+.*.env
+
+# devspace ignores
+.devspace
+
+# node
```

Comment:
This is covered in `simulators/vehicles/train/config/.gitignore`

Reply:
We will need this ignores multiple times. I will remove the subfolder .gitignore on the next PR.

Reply:
ok